lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -31,12 +27,24 @@ export default function AssignmentActivity() {
 
   return (
     <Card>
-      <CardHeader fullWidth>
-        <CardTitle tagName="h2" data-testid="title">
+      <CardHeader fullWidth classes="flex-col !gap-x-0 !items-start">
+        {assignment.data && (
+          <div className="mb-3 mt-1 w-full">
+            <DashboardBreadcrumbs
+              links={[
+                {
+                  title: assignment.data.course.title,
+                  href: urlPath`/courses/${String(assignment.data.course.id)}`,
```

Comment:
Is the `String` needed here for type checking?

Reply:
Yes

Reply:
My preferred way to cast numbers to string is usually by doing ``${theNumber}``, but I think nesting string interpolations is more confusing, so I used `String(theNumber)` here instead.